sw/device/tests/dif_gpio/meson.build

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+gtest_inc = include_directories(
+  '/tools/googletest/gtest',
```

Comment:
Have you looked at what it would take to vendor these?

Reply:
It seems like we can use meson wrap. I will update this PR once I get it working.

Reply:
https://wrapdb.mesonbuild.com/ is down.